### PR TITLE
Add a login panel to the RAD homepage

### PR DESCRIPTION
**Pull request by munckymagik**
# Rationale

We will now have many more users visiting this page to sign-in rather than to register. Therefore we need to make it possible to sign-in more easily from the homepage.

# Screenshots

## On desktop:

screen shot 2015-07-29 at 13 52 14

## On mobile:

**Note:** Side column appears above main column on small screens

screen shot 2015-07-29 at 13 53 31

### CODE REVIEW

File: app/views/principals/pre_qualification_form.html.erb [hunk 1]
```diff
@@ -1,119 +1,129 @@
-<div class="l-constrained">
```
Comment (munckymagik) [inline app/views/principals/pre_qualification_form.html.erb]: This is a terrible diff. Essentially:

* The original content has been **indented** and placed into an `l-2col-main`
* A new `<section>` has been added before it to render the `sign_in_panel` partial
* A new `l-1col` section has been added to place the introductory elements evenly above the columns below.

Any questions, just ask.

**Comment by benbarnett** [comment]
LGTM :+1:

**Comment by neoeno** [comment]
:+1:

**Comment by munckymagik** [comment]
@neoeno can you review the "Fix url matcher ..." commit please?

**Comment by neoeno** [comment]
seems good :+1:

**Pull request merged**